__tests__/main.test.ts

```diff
@@ -3,76 +3,172 @@ import * as process from 'process'
 import * as cp from 'child_process'
 import * as path from 'path'
 import * as rimraf from 'rimraf'
-import { downloadLatest, install } from '../src/restore'
```

Comment:
Remove the import of `restore`, to avoid that it automatically executes the `run()` function, which the test has no control over. It is also conflicting with later invocations of the restore action, because those will find that the symlinks are already set up and fail.